recipes/harbor/recipe.yaml

```diff
@@ -0,0 +1,73 @@
+context:
+  name: harbor
+  version: "0.3.0"
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/choderalab/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: "71e3d0b324d3132a8aa8adaed585a4d8367a048c925e8b905ba00735a4d4be8c"
+
+build:
+  script: SETUPTOOLS_SCM_PRETEND_VERSION=${{ version }} python -m pip install . -vv
```

Comment:
This fails on windows, but is ok since this is `noarch: python` and will only build on linux.